spid_cie_oidc/onboarding/views.py

```diff
@@ -100,4 +101,12 @@ def onboarding_registration (request):
     return render(request, 'onboarding_registration.html', context)
 
 def onboarding_entities (request):
-    return render(request, 'onboarding_entities.html')
+    entity_list = [
```

Comment:
Great

Please create the model where the registration in form wrote the data.
The we'll have a real query instead of this static list